conans/test/functional/generators/cmake_find_package_multi_test.py

```diff
@@ -11,6 +11,91 @@
 from conans.util.files import load
 
 
+@pytest.mark.tool_cmake
+class TestCreateModuleOfficialCMakeTargets:
+
+    consumer = textwrap.dedent("""
+        from conans import ConanFile, CMake
+
+        class Conan(ConanFile):
+            name = "consumer"
+            version = "1.0"
+            settings = "os", "compiler", "build_type", "arch"
+            exports_sources = ["CMakeLists.txt"]
+            generators = "cmake_find_package_multi"
+            requires = "hello/1.0"
+
+            def build(self):
+                cmake = CMake(self)
+                cmake.configure()
+        """)
+
+    def test_global_alias(self):
+        client = TestClient()
+
+        conanfile = textwrap.dedent("""
+            import os
+            from conans import ConanFile, CMake, tools
+
+            class Conan(ConanFile):
+                name = "hello"
+                version = "1.0"
+                settings = "os", "arch", "compiler", "build_type"
+
+                def package_info(self):
+                    self.cpp_info.set_property("aliases", ["hello"], "cmake_find_package_multi")
```

Comment:
The property name is probably not enough:

- Properties are designed to be generic for multiple generators. In most cases, ``package_info()`` will not specify the generator name.
- If this is intended for the CMake build system, something like ``cmake_target_alias`` should do the work.

Reply:
I don't see why alias properly couldn't be generic for several generators.
so far we only needed it in cmake, but theoretically, any build system providing aliases and targets could benefit from it (premake, b2, qbs, etc)

Reply:
You are specializing the call for ``self.cpp_info.set_property("aliases", ["hello"], "cmake_find_package_multi")`` for the ``cmake_find_package_multi`` generator. This shouldn't be the case, the line should be something like:

```
self.cpp_info.set_property("aliases", ["hello"])
```

Then, IMO, it reads too generic, given that there are not known build systems that will use this information, and the information there, by its format pkg::component might not be used by other build systems. So something like ``cmake_target_alias`` makes more sense.

Reply:
I don't know, the property might be generic, but the value of the property might be different for the different generators.
we already know e.g. 'name` and `filename` are generic properties, but different generators use different values, right?
for instance, name in cmake might be `Boost`, but in pkg-config it's `boost`.

Reply:
Exactly: ``name`` and ``filename`` are legacy, to be deprecated by the new properties: ``cmake_target_name``, ``cmake_file_name``. Same pattern to be used.

Reply:
okay, I'll change then